app/lua53/host/Makefile

```diff
@@ -23,6 +23,20 @@ else
   # MAKEFLAGS += --silent -w
 endif  # $(V)==1
 
+
+# Validate LUA setting
+ifeq ("$(LUA)","")
```

Comment:
Same as above. Currently empty/unset means 51. This would need to be adapted when we switch the default to Lua 53

Reply:
Same, the path suggests 5.3 already.